File: benchmark/src/nodejs/strrev.js
const n=50000,s="a".repeat(n),start=Date.now();
let buf=[];for(let i=n-1;i>=0;i--)buf.push(s[i]);
let r=buf.join('');const end=Date.now();
console.log('Result: '+r[0]+'/'+r.length);console.log('Time: '+Math.floor(end-start)+'ms')

File: benchmark/src/lua/strrev.lua
local n=50000;local s=string.rep("a",n);local start=os.clock()
local buf={};for i=#s,1,-1 do buf[#buf+1]=s:sub(i,i)end;local r=table.concat(buf)
local finish=os.clock();print("Result: "..r:sub(1,1).."/"..#r);print(string.format("Time: %.0fms",(finish-start)*1000))